rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2492,6 +2492,10 @@ def test_upgrade_db_43_to_44(user_data_dir, messages_aggregator):
             (bad_address, None, 'yabirgb.eth'),
             (bad_address, None, 'yabir.eth'),
         }
+        cursor.execute(
```

Comment:
hmmm even if it's a test please let's keep clean practises. We never open a cursor for writing longer than needed. Undo the previous change and open another write cursor only for this here.